packages/plugin-workflow-approval/src/client/common/constants/approval-initiation-status-options.ts

```
@@ -1,46 +1,55 @@
   },
   {
     value: APPROVAL_INITIATION_STATUS.ERROR,
-    label: 'ERROR',
+    label: tval('ERROR'),
     color: '#FF0000',
   },
 ];
+
+export const approvalInitiationStatusMap = approvalStatusEnums.reduce(
+  (configMap, options) => ({
+    ...configMap,
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)